frontend/coprs_frontend/coprs/logic/complex_logic.py

```diff
@@ -203,7 +203,7 @@ def get_copr(user_name, copr_name, **kwargs):
             raise ObjectNotFound(
                 message="Project {}/{} does not exist.".format(
                     user_name, copr_name
-                )
+                ),encode()
```

Comment:
Typo, comma instead of a dot

> coprs_frontend/coprs/logic/complex_logic.py:207:13: E0001[syntax-error]: Parsing failed: 'positional argument follows keyword argument (<unknown>, line 207)'